exporter/sentryexporter/sentry_exporter.go

```diff
@@ -74,7 +75,9 @@ func (s *SentryExporter) pushTraceData(_ context.Context, td pdata.Traces) error
 
 			spans := ils.Spans()
 			for k := 0; k < spans.Len(); k++ {
-				sentrySpan := convertToSentrySpan(spans.At(k), library, resourceTags)
+				otelSpan := spans.At(k)
+				sentrySpan := convertToSentrySpan(otelSpan, library, resourceTags)
+				convertEventstoSentryExceptions(otelSpan.Events(), sentrySpan, &exceptionEvents)
```

Comment:
I would prefer if the mutated arg is the first one passed in, makes it more apparent that the function’s side effect is changing a passed in arg.

Reply:
Will address this in next commit.